src/Microsoft.AspNetCore.Sockets/ServerSentEvents.cs

```diff
@@ -21,6 +21,7 @@ public ServerSentEvents(Connection connection)
             context.Response.ContentType = "text/event-stream";
             context.Response.Headers["Cache-Control"] = "no-cache";
             context.Response.Headers["Content-Encoding"] = "identity";
+            await context.Response.Body.FlushAsync();
```

Comment:
Yes. Without that SSE does not quite work - i.e. opened event on EventSource only is sent with the first message,